src/OrchardCore/OrchardCore.Search.Elasticsearch.Core/ElasticsearchConstants.cs

```diff
@@ -2,7 +2,5 @@ namespace OrchardCore.Search.Elasticsearch;
 
 public class ElasticsearchConstants
 {
-    public const string StandardAnalyzer = "standard";
```

Comment:
Non used constant

Reply:
I think this one should be kept still as it is relevant to have it if someone wants to use it instead of a userDefined analyzer.

Reply:
I changed my mind. Ok to remove it.

Reply:
If like that I don't think both constants should have the same value, doesn't make sense